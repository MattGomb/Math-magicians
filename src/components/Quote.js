import React from 'react';

function Quote() {
  return (
    <section className="quote-container">
      <div className="quote">Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston</div>
    </section>
  );
}

export default Quote;
