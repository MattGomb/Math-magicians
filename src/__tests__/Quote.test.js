import Quote from '../components/Quote';
import React from 'react';
import renderer from 'react-test-renderer';

describe('the calculator snapshot and react testing library', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
