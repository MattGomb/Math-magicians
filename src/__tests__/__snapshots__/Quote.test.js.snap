// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the calculator snapshot and react testing library renders correctly 1`] = `
<section
  className="quote-container"
>
  <div
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </div>
</section>
`;
